wg-batch/charter.md

```diff
@@ -19,25 +19,25 @@ providers.
     that benefit batch use cases such as HPC, AI/ML, data analytics and CI.
   - Primitives for job-level queueing, not limited to the k8s Job resource. Long-term,
     this could include multi-cluster support.
-  - Primitives to control and maximize utilization of resources in fixed-size clusters (on-prem) and elastic clusters (cloud).
+  - Primitives to control and maximize utilization of resources in fixed-size clusters 
+    (on-prem) and elastic clusters (cloud).
   - Runtime and scheduling support for specialized hardware (GPUs, NUMA, RDMA, etc.)
-- To congregate leads and users from different external projects and user groups
-  (CNCF TAGs, conferences and workshops) in the batch ecosystem to gather requirements,
-  validate designs and encourage reutilization of core kubernetes APIs.
+- To reduce fragmentation in the k8s batch ecosystem: congregate leads and users from 
+  different external and internal projects and user groups (CNCF TAGs, k8s working grouping
```

Comment:
working group?

I think the group is called Topology Aware Scheduling. However, it's not a WG under k8s. @swatisehgal what's the proper way of calling it?

Reply:
changed to subproject

Reply:
Yep, we have been referring to our group as Topology Aware Scheduling working group but it is not a formal WG under k8s so we can call it Topology Aware Scheduling subproject/team/task force. I think @ahg-g  already made updates to the text so we are good!